### PR TITLE
Amended project summary to include TA's recommendations from feedback…

### PR DESCRIPTION
**Pull request by Radascript**
Edited ReadMe file based on feedback.

### CODE REVIEW

**Review by showcy — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,30 +2,31 @@
...
+
+At the hypothesis test, we identified the difference in the two medians of **-56 days**, with a corresponding *p-value* of **0.0328**. The indigenous group was found to have shorter sentence lengths than the non-indigenous group. As this p-value was smaller than the significance level, therefore there was statistically significant evidence to reject the null hypothesis that stated that there is no difference in the median sentence lengths between the two groups. Considering the large sample size, there is a small chance that the result is affected by the Type II error. However, we have to be mindful of the small effect size. This may raise some concern regarding the practical implications of the study, but we believed it was important not to miss any existing effect. We concluded that in our case, minimizing Type II error was more important than Type I error.
```
Comment (miyer26) [inline README.md]: Current: 'As this p-value was smaller than the significance level, therefore there was statistically significant evidence to reject the null hypothesis that stated that there is no difference in the median sentence lengths between the two groups.'

Proposed: 'As this p-value was smaller than the significance level, there was statistically significant evidence to reject the null hypothesis that stated that there is no difference in the median sentence lengths between the two groups.'

File: doc/sentence_length_diffs_inference_report.Rmd [hunk 2]
```diff
@@ -25,85 +25,86 @@ ci_95 <- readRDS(paste(params$results_dir_path,"/ci_95.rds", sep=""))
...
+The median was selected as the measure of central tendency as the distribution of sentence length for both groups were skewed. Subsequently, a permutation test under the null model was carried out computationally with a significance level of **0.05**. The null hypothesis stated that there was no difference in the population medians in sentence lengths between indigenous and non-indigenous inmates. The alternative hypothesis stated that there was a difference in the population medians in sentence length between indigenous and non-indigenous offenders. The resulting sample difference in the two medians was `r diff_medians`, with a corresponding p-value of `r p_value`. The indigenous group was found to have shorter sentence lengths than the non-indigenous group. As this $p$-vaule was smaller than the significance level, there was statistically significant evidence to reject the null hypothesis that stated that there was no difference in the median sentence lengths between the two groups.
```
Comment (miyer26) [inline doc/sentence_length_diffs_inference_report.Rmd]: Should we be a little more specific and perhaps say: 
The median was selected as the measure of central tendency as the distribution of sentence length for both groups were _right_ skewed.

File: doc/sentence_length_diffs_inference_report.Rmd [hunk 3]
```diff
@@ -25,85 +25,86 @@ ci_95 <- readRDS(paste(params$results_dir_path,"/ci_95.rds", sep=""))
...
+Looking at the box-plots, we noted the large overlap in the confidence intervals between the two groups, which at a glance indicated that a statistically significant difference was unlikely.
 
 ```{r boxplots, fig.width=10, fig.height=3, fig.cap = "Figure 2. The boxplot of the sentence length of offenders by racial group. The marker at the center of the box corresponds to the median.", out.width='100%'}
 
 knitr::include_graphics('../results/eda-box_plots.png')
 ```
 
-We had a large enough sample size to apply a permutation test as the toal sample size was `r format(counts$Count[1] + counts$Count[2])`. We carried out the permutation test with r N_REPEATS under the null hypothesis, computing and comparing the median for each of the two groups.
+We had a large enough sample size to apply a permutation test as the total sample size was `r format(counts$Count[1] + counts$Count[2])`. We carried out the permutation test with `r N_REPEATS` under the null hypothesis, computing and comparing the median for each of the two groups.
```
Comment (miyer26) [inline doc/sentence_length_diffs_inference_report.Rmd]: I do not think the `r N_REPEATS` is parameterized so I think it is better to replace it with the raw value _5000_.

**Review by miyer26 — CHANGES_REQUESTED** [review]
Thank you for the review and write-up Rada. There are a couple of comments that I've written but otherwise everything looks great!

**Pull request merged**